docs/configuration-process.asciidoc

```diff
@@ -51,12 +50,6 @@ Defaults to 30 seconds.
 Maximum duration in seconds before releasing resources when shutting down the server.
 Defaults to 5 seconds.
 
-[[concurrent_requests]]
```

Comment:
As I learned yesterday, we should add removed blocks to the `redirects.asciidoc` section to ensure all links still work.